tests/integration/test_services_audio_preprocessor_analyzer_organizer.py

```
@@ -0,0 +1,1170 @@
+        with patch("subprocess.run") as mock_run:
+            mock_run.return_value = MagicMock(returncode=0)
+            preprocessor = AudioPreprocessor(config=cfg)
+
+        assert preprocessor.config.sample_rate == 44100
+        assert preprocessor.config.channels == 2
+
+    def test_ffmpeg_not_found_logs_warning(self) -> None:
+        from file_organizer.services.audio.preprocessor import AudioPreprocessor
+
+        with patch("subprocess.run", side_effect=FileNotFoundError):
+            # Should not raise – just logs a warning
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)